src/apiProxy/apiConfigCreators.js

```diff
@@ -30,6 +30,34 @@ export function group(params) {
 }
 
 /**
+ * group communication endpoints
+ *
+ * accepts an `method` property in params
+ */
+export function mugcomm(params) {
+	const path = 'communications';
+	const { urlname, conversationId } = params;
```

Comment:
Since all the endpoints start with `${urlname}/${path}`, and the remaining path components are essentially pre-defined in the `params`, you can be DRYer here by writing the endpoint more directly and only having `method` values of `followers` and `suggested_members`:

```js
const { urlname, conversationId, method } = params;

const endpoint = [
  urlname,
  'conversations',
  conversationId,
  method
]
.filter(c => c)  // only include non-empty values
.join('/');  // create a url path string

return {
  endpoint,
  params,
};
```